spec/controllers/sites_controller_spec.rb

```diff
@@ -52,6 +52,23 @@
       end
     end
 
+    context "site with existing favicon" do
+      before do
+        FaviconUploader.enable_processing = false
+        file = fixture_file_upload('/images/favicon.png', 'image/png')
+        Site.instance.update(favicon: file)
+        FaviconUploader.enable_processing = true
+      end
+
+      it "#update with remove_favicon deletes a favicon" do
+        expect(Site.instance.favicon?).to be true
+        post :update, params: { id: Site.instance.id, remove_favicon: 'Remove favicon' }
+        expect(response).to redirect_to('/admin/appearance?locale=en')
```

Comment:
I don't _think_ this is a problem, but if somehow the locale was changed in the test env, would this test fail since it specifies `?locale=en`?

Reply:
If the locale is changed in the test, then you would need to update the path here. this pattern is used throughout this test